src/components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,105 @@
+import './MovieForm.css'
+import React from 'react'
+import { MovieGenres } from '../../common/constants/constants.ts'
+
+type MovieFormProps = {
+    initialInfo?: {
+        title: string
+        releaseDate: string
+        movieUrl: string
+        rating: number
+        runtime: number
+        overview: string
+    }
+    onSubmit: (formData: { [p: string]: FormDataEntryValue }) => void
+}
+
+export function MovieForm({ initialInfo, onSubmit }: MovieFormProps) {
+    function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+        event.preventDefault()
+        const formData = Object.fromEntries(new FormData(event.currentTarget))
+        onSubmit(formData)
+    }
+
+    return (
+        <form onSubmit={handleSubmit} aria-label="form">
+            <div className="movie-form">
```

Comment:
I see you switched to not using curly braces for `className`, but what about other props? It would be awesome if you kept consistency throughout all the props and all the components. BTW, you don't have to do this manually, `eslint` can help you with automatic detection and fixing. Check out the rule: 
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-literals.md